src/corecel/sys/detail/TrackEvent.perfetto.hh

```diff
@@ -0,0 +1,32 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file corecel/sys/detail/TrackEvent.perfetto.hh
+//! \brief Define perfetto track event categories
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <perfetto.h>
+namespace celeritas
+{
+namespace detail
+{
+// Global category name for celeritas
+// TODO: add more categories for fine-grained control of events to record
+constexpr auto* perfetto_track_event_category{"Celeritas"};
```

Comment:
Doesn't this need a static, inline, or a `.cc` file?

Reply:
Right, I forgot inline. I don't think static is needed though.